docs/community/journey.md

```diff
@@ -0,0 +1,82 @@
+# Begin your Journey
+
+First, thanks for taking your precious time to help us ensure testnet is all it can be. 🙏 
+
+## Features and setting expectations
+
+In this release, you will see the following features:
+
+- Privacy across the entire lifecycle of your smart contract by encrypting specific information / API requests considered sensitive, for example, eth_getBalance. The full list of encrypted requests and responses is below.
+
+https://docs.obscu.ro/wallet-extension/handling-sensitive-data.html#sensitive-json-rpc-api-methods
+
+* An Ethereum-like Layer 1. Soon Obscuro Testnet will be hooked into a well-known Ethereum Testnet.
+
+* Straightforward integration with MetaMask using a Wallet Extension binary run locally to hook your MetaMask into Obscuro Testnet.
+
+https://docs.obscu.ro/wallet-extension/wallet-extension.html
+
+* A rudimentary block explorer called ObscuroScan. For Testnet ONLY, this shows the details of transactions in a decrypted format by default to make it easier to understand what is happening on Testnet.
+
+http://testnet-obscuroscan.uksouth.azurecontainer.io/ 
+
+Current Limitations:
+
+We're in our first release, and much more is yet to come. However, some of the current limitations you can expect:
+
+* The gas feature is not implemented yet, nor is native value transfer.
+
+* There is no streaming of events. This will limit how your UI updates based on events being listened to.
+
+* Testnet might crash unexpectedly as sharp edges are smoothed away and edge cases fixed.
+
+* Expect to lose all data when Testnet crashes or is restarted. 
+
+## Challenges
+
+Below are a set of challenges to get you started. For each challenge you successfully complete, you'll be rewarded task points which get transformed into OBX tokens as per the [Tokenomics](https://github.com/obscuronet/obscuro-project/wiki/Tokenomics)
+
+### Warmup - 20 points
+Get set up and deploy a simple contract.
+* Configure the Wallet extension and Metamask following the instructions [here](https://docs.obscu.ro/wallet-extension/wallet-extension.html)
+* Try deploying any contract (e.g. the default Storage contract). Remember to select Injected Provider as your Environment ![Step 1 screenshot](https://images.tango.us/public/screenshot_017a4a1c-b655-4eda-8c6e-7a298e1faa70.png?crop=focalpoint&fit=crop&fp-x=0.0945&fp-y=0.1827&fp-z=2.5115&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=2027%3A1103)
```

Comment:
"If using Remix..." or something like "we recommend using Remix" or "this has been tested using Remix"